tests/lint/check_file_type.py

```diff
@@ -164,8 +164,7 @@ def copyright_line(line):
     # so that the copyright detector won't detect the file itself.
     if line.find("Copyright " + "(c)") != -1:
         return True
-    if (line.find("Copyright") != -1 and
-            line.find(" by") != -1):
+    if line.find("Copyright") != -1 and line.find(" by") != -1:
```

Comment:
# break pattern into two lines to avoid false-negative check
spattern1 = "Copyright"
if line.find(spattern1) != -1 and line.find("by") != -1: